code/index.html

```diff
@@ -1,37 +1,100 @@
-      </article>
-      <article class="weapon-deck" id="weaponCard">
-        <p>WEAPON</p>
-        <p id="weaponName"></p>
-        <p id="weaponWeight"></p>
-      </article>
-      <article class="room-deck" id="roomCard">
-        <p>ROOM</p>
-        <p id="roomName"></p>
-      </article>
-    </section>
-
-    <h1 id="mystery"></h1>
-
-    <button>REVEAL THE CRIME</button>
-
-    <script src="clue.js"></script>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="./style.css" />
+  <link rel="icon" type="image/png" href="./images/favicon.png">
+  <title>SmoothDay - Smoothies for all days</title>
+</head>
+
+<body>
+
+<!-- HEADER -->
+  <header>
+    <h1>SmoothDay</h1>
+    <a href="#">
+      <div class="hamburger">
+        <div class="hamburger--stroke"></div>
+        <div class="hamburger--stroke"></div>
+        <div class="hamburger--stroke"></div>
+      </div>
+    </a>
+  </header>
+
+<!-- INTRO + FAVORITE -->
+  <section class="intro">
+    <p class="intro--text">Start off your day with a smoothie of <span class="italic">our</span> choice.</p>
+
+    <div class="intro--main-button">
+      <button class="main-button" onclick="randomizeSmoothie()">Randomize today's mix &#8594;</button>
+    </div>
+
+    <div class="intro--buttons">
+      <div class="favorite-fruit--form-box">
+        <form>
+          <label for="favorite-fruit" class="favorite-fruit--label">or, if you want to, select your favorite fruit*: </label>
+          <select onchange="selectFavorite(value)" class="favorite-fruit--select">
+           <option value="Select">Select Fruit</option>
+           <option value="Apple">Apple</option>
+           <option value="Banana">Banana</option>
+           <option value="Mango">Mango</option>
+           <option value="Orange">Orange</option>
+           <option value="Raspberry">Raspberry</option>
+           <option value="Lemon">Lemon</option>
+         </select>
+        </form>
+        <p class="favorite-fruit--comment">*This will be chosen more frequently in the randomizer</p>
+      </div>
+
+      <button>Manage your inventory &#8594;</button>
+    </div>
+  </section>
+
+
+<!-- INGREDIENTS/MAIN -->
+  <section class="ingredients-headers"> 
+    <div id="fruit-header" class="main-fruit-header">Main Fruit/berry</div>
+    <div id="mixer-header" class="mixer-header">Mixer</div>
+    <div id="superbooster-header" class="super-booster-header">Super Booster</div>
+  </section>
+
+  <section class="ingredients">
+    <div id="main-fruit-box" class="main-fruit-box"></div>
+    <div id="mixer-box" class="mixer-box"></div>
+    <div id="superbooster-box" class="superbooster-box"></div>
+  </section>
+  
+  <section class="ingredients--randomizer-buttons">
+    <button onclick="pickMainFruit()">Randomize main fruit/berry &#8594;</button>
```

Comment:
Food for thoughts: my first instinct was to click the images. Did you think about putting an event handler on the images instead of having a button?